src/PValueBasedFeatureSelector.jl

```diff
@@ -0,0 +1,110 @@
+"""
+`PValueBasedFeatureSelector` has the following fields:
+
+* `k::Int64` - Select top `k` features with the highest correlation to target
+  variable. You could ignore this by specifying k <= 0. This defaults to 0.
+* `threshold::Float64` - Select features with correlation less than or equal to
+  threshold. Note that, in P-value, a feature is considered important when the value
+  is close to 0. To ignore, simply set threshold to 0 (default behavior).
+* `method::Symbol` - Type of test to get the p-value. The available option are
+  `:FTest` and `:ChiSq`. `:ChiSq` expects the input and target are in non-negative
+  integers.
```

Comment:
For `method`, you could consider using an actual `Function` here rather than a `Symbol`, and just call that function within `select_features`. Maybe that's best left for later though, depending on how / whether you think you will generalize this.